code-build.js

```diff
@@ -35,13 +35,21 @@ async function build(sdk, params) {
 }
 
 async function waitForBuildEndTime(sdk, { id, logs }, nextToken) {
-  const { codeBuild, cloudWatchLogs, wait = 1000 * 5 } = sdk;
+  const {
+    codeBuild,
+    cloudWatchLogs,
+    wait = 1000 * 30,
+    backOff = 1000 * 15
+  } = sdk;
 
   // Get the CloudWatchLog info
   const startFromHead = true;
   const { cloudWatchLogsArn } = logs;
   const { logGroupName, logStreamName } = logName(cloudWatchLogsArn);
 
+  let errorDetected = false;
+  let errMessage = "";
```

Comment:
I'm not going to block on this,
but I'm wondering two things

1. Why call it errMessage when it is the error?
1. Why have a bool _and_ the value? Why not set the err to `false` and then check its value? (since it is strictly private )

Reply:
@seebees totally valid, and fixed